### PR TITLE
Update Observability readme

### PR DESCRIPTION
**Pull request by katyamust**
Rearrange sections for better readability and make it easier discoverable for the reader. 
Extracted sections to separate files. 

Overall made readme shorter with directions to look at for specifics in separate file.  

No content change.

### CODE REVIEW

File: observability/readme.md [hunk 1]
```diff
@@ -1,132 +1,33 @@
...
-
-### Azure DevOps Pipelines Reporting with Power BI
-
-The [Azure DevOps Pipelines Report](https://github.com/Azure-Samples/powerbi-pipeline-report) contains a [Power BI](https://docs.microsoft.com/en-us/power-bi/fundamentals/power-bi-overview) template for monitoring project, pipeline, and pipeline run data from an Azure DevOps (AzDO) organization.
-
-This dashboard recipe provides observability for AzDO pipelines by displaying various metrics (i.e. average runtime, run outcome statistics, etc.) in a table. Additionally, the second page of the template visualizes pipeline success and failure trends using Power BI charts. Documentation and setup information can be found in the project README.
+- [Observability in Machine Learning](ml-observability.md)
```
Comment (NarmathaBala) [inline observability/readme.md:31]: ML and Delivery pipelines don't really fall under the "Recipes" section. What do you think about moving these two to "Facets of Observability" - in reality, these are explaining about how to observe in a particular component, say delivery pipelines or in ML. I think moving these into Recipes gives an impression that these are 'show and tells' of how to do observability. Thoughts?

File: observability/readme.md [hunk 2]
```diff
@@ -1,132 +1,33 @@
...
-    - Additionally, failures in **dependent services** should be propoagated up each level of the stack by the healthcheck.
+- [Dashboards and Reporting](pillars/dashboard.md)
 
-1. **Faults, crashes, and failures** are logged as discrete events. This helps engineers identify problem area(s) during failures.
-1. Ensure logging configuration (eg: setting logging to "verbose") can be controlled without code changes.
-1. Ensure that **metrics** around latency and duration are collected and can be aggregated.
-1. Start small and add where there is customer impact. [Avoiding metric fatigue](pitfalls.md#metric-fatigue) is very crucial to collecting actionable data.
-1. It is important that every data that is collected contains relevant and rich context.
-1. Personally Identifiable Information or any other customer sensitive information should never be logged. Special attention should be paid to any local privacy data regulations and collected data must adhere to those. (ex: GPDR)
+## Tools and best practices
```
Comment (NarmathaBala) [inline observability/readme.md:23]:
```suggestion
## Tools, Patterns and Recommended Practices
```

File: observability/readme.md [hunk 3]
```diff
@@ -1,132 +1,33 @@
...
 
-Read more [here](pitfalls.md) to understand what to watch out for while designing and building an observable system.
+- [Tooling](tools/readme.md)
```
Comment (NarmathaBala) [inline observability/readme.md:25]: Tooling and Patterns

File: observability/readme.md [hunk 4]
```diff
@@ -1,132 +1,33 @@
...
+- Provide holistic view of the _application health_.
+- Help measure _business performance_ for the customer.
+- Measure _operational performance_ of the system.
+- Identify and _diagnose failures_ to get to the problem fast.
 
-1. Provide holistic view on the health of the application.
-2. Help measure business performance for the customer.
-3. Measure operational performance of the system.
-4. Identify and diagnose failures to get to the problem fast.
+## Getting started
```
Comment (NarmathaBala) [inline observability/readme.md:10]: Not quite sure of this header. This is a common practice across all fundamentals?

**Review by NarmathaBala — COMMENTED** [review]
@katya, these changes largely looks good. Thank you for your contribution. Leaving it open to hear your thoughts on one open question

File: observability/readme.md (hunk 1 above)
Comment (ivaramme) [reply observability/readme.md:31]: I like the term "Facets" +1 on that.

Comment (katyamust) [reply observability/readme.md:29]: I like the term "Fasets" too! Updated  and left recipes under this category

File: observability/readme.md (hunk 4 above)
Comment (katyamust) [reply observability/readme.md]: I removed it - I was trying to put under same umbrella pillar and insights. I now moved them to upper level.

File: observability/readme.md (hunk 3 above)
Comment (katyamust) [reply observability/readme.md]: Updated

**Review by NarmathaBala — APPROVED** [review]
These changes look good, thanks for the content @katyamust

**Review by TessFerrandez — APPROVED** [review]

**Pull request merged**